templates/vault/0/docker-compose.yml.tpl

```diff
@@ -0,0 +1,35 @@
+version: '2'
+volumes:
+  vault-config:
+    driver: ${VOLUME_DRIVER}
+  vault-file:
+    driver: ${VOLUME_DRIVER}
+services:
+  vault-lb:
+    image: rancher/lb-service-haproxy:v0.9.1
+    ports:
+    - ${VAULT_LISTEN_PORT}:8200/tcp
+    - ${VAULT_CLUSTER_PORT}:8201/tcp
+    labels:
+      io.rancher.scheduler.affinity:host_label: lbhost=true
```

Comment:
What do you think if you add a question on `rancher-compose.yml` to be able to set your own host label??
```
-      io.rancher.scheduler.affinity:host_label: lbhost=true
+      io.rancher.scheduler.affinity:host_label: ${HOST_LABEL}
```